spec/index.md

```diff
@@ -866,12 +866,20 @@ It must be contained in the canister ranges of a subnet, otherwise the correspon
 
     -   Otherwise, the call is rejected by the system independently of the effective canister id.
 
--   If the request is an update call to a canister that is not the Management Canister (`aaaaa-aa`) or if the request is a query call, then the effective canister id must be the `canister_id` in the request.
+-   If the request is a query call to the Management Canister (`aaaaa-aa`), then:
+
+    -   If the call is to the `bitcoin_get_balance_query` or `bitcoin_get_utxos_query` method, then the effective canister id for this call must be the bitcoin canister id which is left unspecified (in particular, it depends on the bitcoin network).
```

Comment:
This line hints at the fact that there is a Bitcoin canister.
Note that insinuating that this protocol-level feature is backed by canisters in the interface spec would require the approval of upper management.
It would be better to avoid this path in my opinion.

Reply:
The existence of the Bitcoin canister is publicly revealed in the [docs](https://internetcomputer.org/docs/current/references/bitcoin-how-it-works#components).